src/timer/timer32.rs

```diff
@@ -1,21 +1,19 @@
 //! HAL library for Timer module (Timer32) - MSP432P401R
 pub use embedded_hal::timer::{Cancel, CountDown, Periodic};
+use crate::common::{Split, NotDefined, Defined};
 
 use pac::TIMER32;
 use core::marker::PhantomData;
-use core::mem::transmute;
 
 use crate::clock::Clocks;
 use crate::time::Hertz;
 use crate::timer::*;
 
-const MAX_PRESCALER32: u32 = 0x0100;
+unsafe impl Send for Channel0 {}
+unsafe impl Send for Channel1 {}
```

Comment:
Why is this necessary?

Reply:
A type is Send if it is safe to send it to another thread.